webserver/views/api.py

```diff
@@ -208,9 +96,132 @@ def get_listens(user_id):
         del temp['user_id']
         listen_data.append(temp)
 
-    payload = {}
-    payload['user_id'] = user_id
-    payload['count'] = count
-    payload['listens'] = listen_data
+    return json.dumps({'payload': {
```

Comment:
is this an api endpoint? is it better to use `jsonify` in order to send correct headers